src/Ilios/CoreBundle/Classes/CurriculumInventory/Exporter.php

```diff
@@ -0,0 +1,1044 @@
+<?php
+
+namespace Ilios\CoreBundle\Classes\CurriculumInventory;
```

Comment:
Might be better off in /Service thought I think the line is a little blurry in CoreBundle